docs/dev/Lua API.rst

```diff
@@ -5589,44 +5589,60 @@ TextArea Functions:
 * ``textarea:scrollToCursor()``
 
     Scrolls the text area view to ensure that the current cursor position is visible.
-    This is useful for automatically scrolling when the user moves the cursor
-    beyond the visible region of the text area.
+    This happens automatically when the user interactively moves the cursor or
+    pastes text into the widget, but may need to be called when ``setCursor`` is
+    called programmatically.
 
 * ``textarea:clearHistory()``
 
-    Clear undo/redo history of the widget.
+    Clears undo/redo history of the widget.
 
-Functionality:
+Functionality
+-------------
```

Comment:
welcome to the wonderful world of restructured text header underline rules. Currently, this section is interpreted like this:
![image](https://github.com/user-attachments/assets/c7010994-b79b-44fc-bbcf-f8dffb88b7f7)
that is, `Functionality` is a peer of `TextArea class` and `Cursor Behavior` is a *higher level* header than `TextArea class`.

You need to scan through the existing headers and discover what underline style is used for what level of header. It's different for every file, and is determined by whatever the file uses first for that particular header depth.

The indentation for list continuation lines is also apparently different from what sphinx expects. You can see how this PR's text is currently rendered here: https://dfhack--4995.org.readthedocs.build/en/4995/docs/dev/Lua%20API.html#textarea-class

You can get to that link by clicking on the "details" link for the ReadTheDocs build action: 
![image](https://github.com/user-attachments/assets/f909925b-fb9d-4bbf-b5fc-f6667ec0c24e)